function-sidecar/pkg/dispatcher/grpc/grpc.go

```diff
@@ -147,3 +219,27 @@ func toDispatcher(grpc *function.Message) message.Message {
 	}
 	return message.NewMessage(grpc.Payload, dHeaders)
 }
+
+func unmarshallFactory() WindowingStrategyFactory {
+	var w v1alpha1.Windowing
+	json.Unmarshal([]byte(os.Getenv("WINDOWING_STRATEGY")), &w)
+	// TODO: this can be done reflectively, once there are more alternatives.
+	// Also, check there is at most one key with a non-zero value
+	if w.None {
+		log.Printf("Will use 'infinite' windowing strategy\n")
```

Comment:
I would log that it will *not* use a windowing strategy (i.e. unbounded stream). I'd avoid the term 'infinite' in favor of consistency around bounded vs. unbounded terminology.